### ISSUE TITLE
Pomo timer shows incorrect time left

### ISSUE THREAD

**Issue by A-raj468** [epwalsh/pomo.nvim#31]
## The issue
When I set a pomo timer of more than 60s, when displaying the timer additional 30m are added to the minute fields.
Eg: If I set a timer for 20m, it will show 50m, if I set timer for 1m 15s, it shows 31m 15s, ets. (One example included in the screenshot below)
The actual timer works correctly. Only the display, both in noice display box and lualine, are wrong.

## Config(using lazy package manager):
```lua
return {
    {
        -- Pomodoro timer
        'epwalsh/pomo.nvim',
        version = '*',
        lazy = true,
        cmd = { 'TimerStart', 'TimerRepeat', 'TimerSession' },
        dependencies = {
            'rcarriga/nvim-notify',
        },
    },
}
```
My nvim config: [nvim-config](https://github.com/A-raj468/nvim-config)
### Setting pomo timer for 75s:
![image](https://github.com/user-attachments/assets/754fec3c-c0ed-4a9f-9fe9-4f7b16f865d7)
### Timer showing 31m 10s
![image](https://github.com/user-attachments/assets/9d5f5ab6-b001-4917-953b-83e31f1e3072)
### Timer goes to 59s once 16s have passed from original timer
![image](https://github.com/user-attachments/assets/11f8c290-ed60-48b8-a0cc-d06d9d305eb0)

### Versions

```console
❯ nvim --version
NVIM v0.10.0
Build type: RelWithDebInfo
LuaJIT 2.1.1713484068
Run "nvim -V1 -v" for more info
❯ nvim --headless -c 'lua print(require("pomo.version"))' -c q
0.7.0
```
My [lazy-lock.json](https://github.com/A-raj468/nvim-config/blob/main/lazy-lock.json) file.

**Comment by tonnycheruiyot** [comment]
I guess the minutes bit was fixed. The issue now is with the hour display. If the time left is beyond 1h, the time left is shown as x+3h, eg, If you set the timer to 1h30m, both the time left is displayed as 4h30m

**Comment by A-raj468** [comment]
For me, both the issues are there minutes showing additional 30m and hour showing additional 5h.

**Comment by A-raj468** [comment]
I realized the issue is that when formatting the time string, in the `util.lua` module the formatting function uses `os.date("%Mm %Ss", time_left)`. This uses the local time zone to display the time(which in my case is UTC+0530, additional 5hr and 30min). This is exactly the additional time added in the display.

Using UTC to format instead of local time zone fixes the issue. Changing the format string in `os.date` as `os.date("!%Mm %Ss", time_left)` forces the os.date module to use UTC instead of local time zone.

I have requested this change in the PR #32. Hopefully the maintainer merges this change after reviewing it.

**Comment by tonnycheruiyot** [comment]
That actually makes sense. I hope the pr gets merged soon.

**Comment by dfenerski** [comment]
Hey, I experience this too on a fresh install. Any progress with the PR?

**Comment by A-raj468** [comment]
Nah, he hasn't replied yet to the PR. For now, I am using my fork. You can also use that if you want.
